openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css

```diff
@@ -465,6 +465,7 @@
     @apply tw-font-sans;
     width: 450px;
     word-break: break-word;
+    z-index: 10001;
```

Comment:
tailwind is deprecated, Advice to use AntD over tailwind and not add or modify anything on tailwind